api/serializers.py

```diff
@@ -28,6 +28,13 @@ class Meta:
                   'week_assignments_capacity', 'wanted_assignments', 'phone_number', 'email', 'parental_consent',
                   'languages']
 
+    def validate_date_of_birth(self, date_of_birth):
```

Comment:
This can be removed, because it's already in the model validators - validate it works and remove it here